includes/class-wc-siftscience-admin.php

```diff
@@ -158,14 +158,25 @@ private function output_settings_stats() {
 			}
 
 			$stats = get_option( WC_SiftScience_Options::STATS, 'none' );
-			if ( 'none' === $stats ) {
+			if ( $stats === 'none' ) {
 				echo '<p>No stats stored yet</p>';
 				return;
 			}
 
-			$stats = json_decode( $stats );
-			$stats = json_encode( $stats, JSON_PRETTY_PRINT );
-			echo '<pre>' . $stats . '</pre>';
+			$stats = json_decode( $stats , true );
+
+			foreach ( array_keys( $stats ) as $outer_v ) {
+										
+				echo '<table><thead>
+					<tr><th colspan="2" style="text-align:left">' . $outer_v . ':</th></tr>
+					</thead><tbody>';
+
+				foreach ( array_reverse( $stats[ $outer_v ] ) as $inner_k => $inner_v ) {
```

Comment:
Why array_reverse?

Reply:
time is four chars long unlike count looks better 
can be changed

Reply:
Ok, it's fine.